homework/generate-sequence/generateSequence.hpp

```diff
@@ -1,7 +1,18 @@
 #pragma once
 #include <vector>
+#include <algorithm>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    if (count <= 0){
+        std::vector<int> vec;
+        return vec;
+    } else {
```

Comment:
If you return variable here, you don't need to use `else if` or `else`

You can return {}. Compiler will automatically construct empty vector.

Reply:
> If you return variable here, you don't need to use `else if` or `else`
> 
> You can return {}. Compiler will automatically construct empty vector.

Thank you for sugestion